### PR TITLE
feat: implemented modal for adding cash expenses

### PR DESCRIPTION
**Pull request by ssbenedek**
### What is the purpose of this PR?

- Created the modal for addig new cash expenses
- Only frontend, not connected to any database, expenses stored in a useState array

### How did you verify your code is stable and works correctly?

- manual testing

### CODE REVIEW

File: components/AddCashExpenseModal.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+  return (
+    <div>
+      <Tooltip arrow title={"Add cash expense"}>
+        <IconButton onClick={openCashModal} sx={{ mr: "0.5rem" }}>
+          <AttachMoneyIcon />
+        </IconButton>
+      </Tooltip>
+      <Modal open={isModalOpen} onClose={handleClose}>
+        <Box sx={style}>
```
Comment (kamka427) [inline components/AddCashExpenseModal.tsx:96]: Is it possible to use TailwindCSS here? Not a problem, but it would be an improvement for maintainability.

**Review by kamka427 — COMMENTED** [review]
Good starting point! Please fill the verification part in the PR description before merge.
Connecting to the DB should be in a separate PR. Also check my comment.

**Review by MilosAlex — APPROVED** [review]
Looks good!
Next time if possible use tailwind and custom atomic components, but its a nicely done and working solution!

**Review by kamka427 — APPROVED** [review]

File: components/AddCashExpenseModal.tsx (hunk 1 above)
Comment (kamka427) [reply components/AddCashExpenseModal.tsx:96]: Good for this PR!

**Pull request merged**